tests/test_data_import.py

```diff
@@ -0,0 +1,435 @@
+        "test_time": 0,
+        "ah_eff": 0,
+        "e_eff": 0,
+    }]
+    result_pd_dfc = pd.DataFrame(data=result_dfc, columns=result_dfc_cols)
+    print("======DFC======")
+    df_print(output_dfc, result_pd_dfc)
+
+    result_dft_cols = ["cycle_index", "cell_id", "cycle_time"]
+    result_dft = [{"cycle_index": 1, "cell_id": 1, "cycle_time": 0}]
+    result_pd_dft = pd.DataFrame(data=result_dft, columns=result_dft_cols)
+
+    print("======DFT======")
+    df_print(output_dft, result_pd_dft)
+
+    assert output_dft.equals(result_pd_dft)
+    assert output_dfc.equals(result_pd_dfc)
+
+
+def test_prepare_maccor_file():
+    maccor_file = "/bas/data/01_raw/default_user/data_set_samples/cycle/MACCOR_example/MACCOR_example.txt"
+    result_path = (
+        "/bas/data/01_raw/default_user/data_set_samples/cycle/MACCOR_example/MACCOR_example.txt_df"
+    )
+    cellpath_df = prepare_maccor_file(maccor_file)
+    assert cellpath_df == result_path
+    assert os.path.exists(cellpath_df)
+    os.remove(cellpath_df)
+    assert ~os.path.exists(cellpath_df)
+
+
+def test_signedCurrent():
+    assert signedCurrent("D", 1) == -1
+    assert signedCurrent("X", 1) == 1
+
+
+def test_read_ornlabuse():
+    cell_id = "S1Abuse"
+    file_path = rawTestDataPath + "abuse-ornl/"
+    output_df = read_ornlabuse(cell_id, file_path)
+    result_df_cols = [
+        "test_time",
+        "axial_d",
+        "v",
+        "axial_f",
+        "temp_1",
+        "temp_2",
+        "temp_3",
+        "temp_4",
+        "temp_5",
+        "temp_6",
+        "cell_id",
+    ]
+    result_df = [
+        {
+            "test_time": 0.000,
+            "axial_d": 0,
+            "v": 4.128,
+            "axial_f": 0,
+            "temp_1": 0.00,
+            "temp_2": 0.00,
+            "temp_3": 0.00,
+            "temp_4": 0.00,
+            "temp_5": 0.00,
+            "temp_6": 0.00,
+            "cell_id": cell_id,
+        },
+        {
+            "test_time": 0.046,
+            "axial_d": 0,
+            "v": 4.137,
+            "axial_f": 1,
+            "temp_1": 0.00,
+            "temp_2": 0.00,
+            "temp_3": 0.00,
+            "temp_4": 0.00,
+            "temp_5": 0.00,
+            "temp_6": 0.00,
+            "cell_id": cell_id,
+        },
+        {
+            "test_time": 0.000,
+            "axial_d": 0,
+            "v": 0,
+            "axial_f": 0,
+            "temp_1": 22.27,
+            "temp_2": 22.12,
+            "temp_3": 22.44,
+            "temp_4": 22.03,
+            "temp_5": 22.08,
+            "temp_6": 22.58,
+            "cell_id": cell_id,
+        },
+        {
+            "test_time": 0.500,
+            "axial_d": 0,
+            "v": 0,
+            "axial_f": 0,
+            "temp_1": 22.23,
+            "temp_2": 22.03,
+            "temp_3": 22.56,
+            "temp_4": 22.07,
+            "temp_5": 22.12,
+            "temp_6": 22.62,
+            "cell_id": cell_id,
+        },
+    ]
```

Comment:
This test is failing for me because the output dataframe from read_ornlabuse does not match your test dataframe. In data_import.py the output structure uses the following values instead:

```suggestion
    result_df_cols = [
        "test_time",
        "axial_d",
        "v",
        "axial_f",
        "pos_terminal_temperature",
        "neg_terminal_temperature",
        "left_bottom_temperature",
        "right_bottom_temperature",
        "above_punch_temperature",
        "below_punch_temperature",
        "cell_id",
    ]
    result_df = [
        {
            "test_time": 0.000,
            "axial_d": 0,
            "v": 4.128,
            "axial_f": 0,
            "pos_terminal_temperature": 0.00,
            "neg_terminal_temperature": 0.00,
            "left_bottom_temperature": 0.00,
            "right_bottom_temperature": 0.00,
            "above_punch_temperature": 0.00,
            "below_punch_temperature": 0.00,
            "cell_id": cell_id,
        },
        {
            "test_time": 0.046,
            "axial_d": 0,
            "v": 4.137,
            "axial_f": 1,
            "pos_terminal_temperature": 0.00,
            "neg_terminal_temperature": 0.00,
            "left_bottom_temperature": 0.00,
            "right_bottom_temperature": 0.00,
            "above_punch_temperature": 0.00,
            "below_punch_temperature": 0.00,
            "cell_id": cell_id,
        },
        {
            "test_time": 0.000,
            "axial_d": 0,
            "v": 0,
            "axial_f": 0,
            "pos_terminal_temperature": 22.27,
            "neg_terminal_temperature": 22.12,
            "left_bottom_temperature": 22.44,
            "right_bottom_temperature": 22.03,
            "above_punch_temperature": 22.08,
            "below_punch_temperature": 22.58,
            "cell_id": cell_id,
        },
        {
            "test_time": 0.500,
            "axial_d": 0,
            "v": 0,
            "axial_f": 0,
            "pos_terminal_temperature": 22.23,
            "neg_terminal_temperature": 22.03,
            "left_bottom_temperature": 22.56,
            "right_bottom_temperature": 22.07,
            "above_punch_temperature": 22.12,
            "below_punch_temperature": 22.62,
            "cell_id": cell_id,
        },
    ]
```